.github/workflows/build.yml

```diff
@@ -669,14 +669,12 @@ jobs:
           path: build
 
   freebsd-cmake-job:
-    runs-on: macos-12
-    if: "false && !contains(github.event.head_commit.message, 'ci/freebsd skip') && !contains(github.event.head_commit.message, 'ci/freebsd/cmake skip')"
+    runs-on: ubuntu-latest
+    if: "!contains(github.event.head_commit.message, 'ci/freebsd skip') && !contains(github.event.head_commit.message, 'ci/freebsd/cmake skip')"
```

Comment:
You forgot to enable them, I've fixed it (but with force push, to squash the commits, hope it is OK)